### PR TITLE
PS-7861 Release Notes for 5.7.35-38 (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
Modified files:
conf.py
release-notes_index
Added file:
Percona-Server-5.7.35-38

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.35-38
+includes all the features and bug fixes available in
+`MySQL 5.7.35 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-4641`: The `innodb_flush_method <https://www.percona.com/doc/percona-server/5.7/scalability/innodb_io.html#innodb_flush_method>`__ documentation needs more explanation.
+* :jirabug:`PS-7715`: Missing documentation on several `Data-At-Rest <https://www.percona.com/doc/percona-server/5.7/security/data-at-rest-encryption.html>`__ variables
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Describe / documeny missing variable for Data-at-rest encryption

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.35-38
+includes all the features and bug fixes available in
+`MySQL 5.7.35 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-4641`: The `innodb_flush_method <https://www.percona.com/doc/percona-server/5.7/scalability/innodb_io.html#innodb_flush_method>`__ documentation needs more explanation.
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Frankly, it's unclear what's been documented. I'd suggest rephrasing to "Extend the description of the innodb flush method with ..."

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-1346`: LP #1163232: Anomaly with ``opt_log_slow_slave_statements``.
+* :jirabug:`PS-1344`: LP #1160436: The ``log_slow_statement`` is called unconditionally
+* :jirabug:`PS-7582`: Segmentation fault with the data masking plugin
+* :jirabug:`PS-1108`: LP #1704163: Changing a column from uncompressed to compressed for JSON crashes the server
+* :jirabug:`PS-2433`: LP #1234346: include a timestamp in the slow query log file when initializing a new file
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Include

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Removed.

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Removed.

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Fixed.

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.7.35-38.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+*Percona Server for MySQL* 5.7.35-38
+================================================================================
+
+:Date: August 18, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.35-38
+includes all the features and bug fixes available in
+`MySQL 5.7.35 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html>`_
+in addition to enterprise-grade features developed by Percona.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-5.7.35-38.rst]: How about linking to these features here?
```suggestion
in addition to `enterprise-grade features <https://www.percona.com/software/mysql-database/percona-server/feature-comparison>`_ developed by Percona.
```

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.35-38.rst]: Added.

**Pull request merged**